csm/conf/post_install.py

```diff
@@ -123,3 +126,28 @@ def create(self):
             Conf.set(const.CSM_GLOBAL_INDEX, f"{const.DEPLOYMENT}>{const.MODE}",
                      const.DEV)
         Conf.save(const.CSM_GLOBAL_INDEX)
+
+    def set_cluster_id(self):
+        """
+        This Function sets cluster_id in csm configuration.
+        :return:
+        """
+        clusterid = Conf.get(const.CONSUMER_INDEX, const.CLUSTERID_KEY)
+        Log.info(f"Setting cluster id:{clusterid}")
+        Conf.set(const.CSM_GLOBAL_INDEX, const.KEY_CLUSTERID, clusterid)
+
+    def set_node_id(self):
+        """
+        This Function Sets node id's for each storage set in csm configuration.
+        :return:
+        """
+        node_ids = []
+        Log.info("Fetching node ids from each storage set.")
+        num_storage_sets = Conf.get(const.CONSUMER_INDEX, const.NUM_STORAGE_SET)
+        for index in range(num_storage_sets):
+            node_ids =  node_ids + Conf.get(const.CONSUMER_INDEX,
+                                            f'cluster>storage_set[{index}]>nodes')
+        Log.info("Setting node id.")
+        Conf.set(const.CSM_GLOBAL_INDEX, const.NUM_NODE_ID, str(len(node_ids)))
+        for index, node_id in enumerate(node_ids):
+            Conf.set(const.CSM_GLOBAL_INDEX, f"{const.KEY_NODE_ID}[{index}]", str(node_id))
```

Comment:
Add a blank line at the end of the file.

Reply:
Addressed